src/python/gi/overrides/BlockDev.py

```diff
@@ -942,7 +942,7 @@ def __dir__(self):
         """Let's make TAB-TAB in ipython work!"""
 
         if self._use_local:
-            items = set(dir(self._mod) + locals().keys())
+            items = set(dir(self._mod) + list(locals().keys()))
```

Comment:
Using `set()` and then union on the two sets would make this look more professional. :wink:

Reply:
Ok...